wlib/LangRef.cpp

```diff
@@ -711,32 +728,34 @@ class CLangRef : public ILangRef, public CUnknown
 	}
 };
 
+class CECLLangRef : public CLangRef
+{
+};
```

Comment:
No, in fact, that revealed a place where I did not update CECLLangRef to CLangRef.